.editorconfig

```diff
@@ -1,6 +1,12 @@
 root = true
 
 [*]
-insert_final_newline = true
 indent_style = space
 indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
I think this changes don't belong to this PR. If true please remove. No problem if you want to open another PR to add this.

Reply:
Ah yes this is from #2734, I will remove from this PR.